javaslang/src/test/java/javaslang/collection/AbstractSeqTest.java

```diff
@@ -1822,6 +1822,18 @@ public void shouldTransposeIfEmpty() {
     }
 
     @Test
+    public void shouldTransposeIfIs1x0() {
```

Comment:
do we need the supressions for the other tests, if apparently they're not needed here :)?